### PR TITLE
Bump rancher-webhook version to v0.3.2-rc3

### PR DESCRIPTION
**Pull request by KevinJoiner**
## Issues: https://github.com/rancher/rancher/issues/39854 and https://github.com/rancher/rancher/issues/39923

### CODE REVIEW

**Comment by KevinJoiner** [comment]
Charts PR: https://github.com/rancher/charts/pull/2288

**Review by crobby — APPROVED** [review]
lgtm once charts PR merges

**Review by maxsokolovsky — APPROVED** [review]

**Comment by KevinJoiner** [comment]
https://github.com/rancher/rancher/pull/39971

**Pull request closed without merging**
